src/main/java/com/inventage/nexusaptplugin/cache/generators/ReleaseGenerator.java

```diff
@@ -69,31 +73,41 @@ public ReleaseGenerator(DebianFileManager fileManager) {
         // Create Releases
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         OutputStreamWriter w = new OutputStreamWriter(baos);
-        
+
         // write date to fix apt-get update on version 1.1.10 or newer
-        w.write("Date: ");
-        w.write(formatDate(new Date()));
-        w.write("\n");
-        
+        Date now = new Date();
+
+        // Compute Valid-Until date
+        GregorianCalendar calendar = new GregorianCalendar();
+        calendar.setTime(now);
+        calendar.add(Calendar.DATE, 7);
+        Date nextWeek = new Date( calendar.getTime().getTime() );
+
+        String hdrfmt = new String("%s: %s\n");
+
+        w.write(String.format(hdrfmt, "Date", formatDate(now)) +
```

Comment:
Most of these fields are optional. I'd rather not include them than have them hardcoded. If we can provide a configuration option for things like Origin or Description, I am okay with including them.